mapping/sustainability.yaml

```
@@ -323,7 +323,7 @@
     mapping: |-
         Project level:
 
-        For each measure add a `ClimateMeasure` object to the `environment.climateMeasures` array, setting it's `.type` according to the matching code in the open climateMeasuresType codelist and mapping the further explanation to its `.description`.
+        For each measure add a `ClimateMeasure` object to the `environment.climateMeasures` array, setting it's `.type` according to the matching code in the open climateMeasures codelist and mapping the further explanation to its `.description`.
 
         If there is an accompanying document publish in `documents`, with `.documentType` set to 'climateMeasures' and include a short description and/or a link to a document providing details. If the details are part of a more general document indicate the relevant section of the document using `.pageStart` and `.pageEnd`.
```

Refer to common operation.

```suggestion
        If there is an accompanying document, [add a project document](../common.md#add-a-project-document) and set `.documentType` to 'climateMeasures'.
```